ctk/common/src/test/kotlin/org/codice/compilance/ReportListener.kt

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright (c) 2019 Codice Foundation
+
+Released under the GNU Lesser General Public License; see
+http://www.gnu.org/licenses/lgpl.html
+*/
+package org.codice.compilance
+
+import io.kotlintest.Description
+import io.kotlintest.TestResult
+import io.kotlintest.extensions.TestListener
+import org.codice.compliance.report.Report
+
+/**
+ * Listener used to reset the Report map for testing purposes before each test
+ */
+object ReportListener : TestListener {
+
+    override fun afterTest(description: Description, result: TestResult) {
```

Comment:
✏️ Overriding `afterTest` but the docs say "before each test". I know you'll get the same effect, but probably better to get the docs and implementation sync'd. I'd change the docs to match.

Reply:
That's my bad. I changed the name when Jeter suggested it but forgot the comment